### DISCUSSION TITLE
New ML models are not uploaded

### DISCUSSION THREAD

**Discussion by utrack** [immich-app/immich#11816] [General]
Hey, piggybacking of the new MLs' announcements here: https://github.com/immich-app/immich/pull/11468
It says that the model `nllb-clip-large-siglip__mrl` is now supported, but there's no such model in the [CLIP collection](https://huggingface.co/collections/immich-app/multilingual-clip-654eb08c2382f591eeb8c2a7).

According to the MR's code (and update dates), `nllb-clip-large-siglip__v1` is a different model - how can I download the mrl one?

**Comment by dvbthien** [comment]
Hi @utrack,
It's here: https://huggingface.co/immich-app/nllb-clip-large-siglip__mrl
When you switch to nllb-clip-large-siglip__mrl, Immich will automatically download the new model.
<img width="882" alt="Screenshot 2024-08-15 at 23 28 21" src="https://github.com/user-attachments/assets/9ddd6114-c16b-47d3-b0b0-328f6317ba5d">

**Comment by mertalev** [comment]
Since there are so many models and many of them are outclassed by other options, the collections are now curated to include a smaller set of recommended models. I updated the multilingual collection to have the `mrl` variants of the NLLB models instead.
